openapi/openapi-snapshots.ytt.yaml

```diff
@@ -83,6 +83,12 @@ paths:
           required: true
           schema:
             type: string
+        - name: wait
```

Comment:
this block needs to be removed given that there is no async. behavior possible on `GET`